### PR TITLE
Teamtailor Trace Added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Request

**Examples:**

https://jobs.amphi.com/
https://careers.airhelp.com/
https://careers.arsenal.com/
https://careers.mpb.com/
https://careers.assessment.aon.com/
https://careers.ro.mdpi.com/

### CODE REVIEW

File: src/technologies/t.json [hunk 1]
```diff
@@ -760,6 +760,20 @@
     },
     "website": "https://www.jetbrains.com/teamcity/"
   },
+  "Teamtailor": {
+    "cats": [
+      101
+    ],
+    "description": "Teamtailor is Recruitment & Marketing Platform and its used to manage and hire top talent.",
+    "icon": "teamtailor.png",
```
Comment (nurbek91) [inline src/technologies/t.json:768]: Teamtailor
svg available

File: src/technologies/t.json [hunk 2]
```diff
@@ -760,6 +760,20 @@
     },
     "website": "https://www.jetbrains.com/teamcity/"
   },
+  "Teamtailor": {
+    "cats": [
+      101
+    ],
+    "description": "Teamtailor is Recruitment & Marketing Platform and its used to manage and hire top talent.",
```
Comment (nurbek91) [inline src/technologies/t.json:767]: Teamtailor is an applicant tracking system, career site and analytics dashboard combined, mobile friendly, all-in-one recruitment platform.

File: src/technologies/t.json [hunk 3]
```diff
@@ -760,6 +760,20 @@
...
+    "icon": "teamtailor.png",
+    "js": {
+      "Teamtailor.Ziggeo": ""
+    },
+    "pricing": [
+      "poa"
+    ],
```
Comment (nurbek91) [inline src/technologies/t.json:774]:
```
"pricing": [
      "mid",
      "recurring"
    ],
```

**Review by nurbek91 — CHANGES_REQUESTED** [review]

File: src/technologies/t.json (hunk 1 above)
Comment (dhavalveera) [reply src/technologies/t.json:768]: can you please tell me from where you get all this svg icons?

**Comment by dhavalveera** [comment]
@nurbek91 @AliasIO = can you please help me here why the validate check is failing? till yesterday everything was working fine

**Comment by AliasIO** [comment]
Looks like a accidentally pasted some text in the wrong place, nothing to do with your changes.  👍

**Comment by dhavalveera** [comment]
> Looks like a accidentally pasted some text in the wrong place, nothing to do with your changes. 👍

OK! Great

**Pull request merged**